Cotton Classing Database File Import Plugin/PublisherDataModel/Model/NDBData.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace CottonClassingPlugin.PublisherDataModel
+{
+    public class NDBData : BaseObject
+    {
+        public List<NDBRecord> NDBRecords { get; set; }
+
+        public NDBData()
+        {
+            NDBRecords = new List<NDBRecord>();
+        }
+    }
+
+    //public class NDBRecord : BaseObject
+    public class NDBRecord
+    {
+        public string GINCode { get; set; }
+        public string GINBale { get; set; }
+        public string DateClassed { get; set; }
+        public string ReportingBasis { get; set; }
+        public string ModuleTrailerNumber { get; set; }
+        public string NumberOfBales { get; set; }
+        public string OfficialColorGrade { get; set; }
+        public string StapleLength { get; set; }
+        public string Micronaire { get; set; }
+        public string Strength { get; set; }
+        public string LeafGrade { get; set; }
+        public string ExtraneousMatter { get; set; }
+        public string Remarks { get; set; }
+        public string HVIColorGrade { get; set; }
+        public string ColorQuadrant { get; set; }
+        public string RD { get; set; }
+        public string B { get; set; }
```

Comment:
Is there a definition for B and RD to be found somewhere? Might be best to either add a comment here or - better - rename this to a more specific name